.travis.yml

```diff
@@ -10,22 +10,14 @@ sudo: required
 addons:
   chrome: stable
 
-script:
-  # Bazel build & test everything. Can't use `bazel build ...` yet, because
-  # javaharness doesn't build on Travis (needs working Android SDK).
-  - >
-    bazel build --noshow_progress --noshow_loading_progress
-    //particles/...
-    //src/...
-    //src_kt/...
-  - >
-    bazel test --noshow_progress --noshow_loading_progress --test_output=errors
-    //particles/...
-    //src/...
-    //src_kt/...
-  - npm run test-with-start
-  - npm run build:rollup
-  - npm --prefix=server test
+# Bazel build & test everything. Can't use `bazel build ...` yet, because
+# javaharness doesn't build on Travis (needs working Android SDK).
+jobs:
+  include:
+    - stage: build
+      script: bazel build --noshow_progress --noshow_loading_progress //particles/... //src/... //src_kt/... && npm run build:rollup
```

Comment:
Can remove the `npm run build::rollup`, I am amidst disabling server/cloud in CI for now.